### DISCUSSION TITLE
Labels for translation-related issues

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6193] [General]
There should be a dedicated label to add to issues related to translations such as new translations, fixes, etc.

The reason is easier search as with the current system, finding an existing issue regarding a specific translation may not be guaranteed if you lack the right search term.
With the label, all issues (At least from the point the label was introduced) could more easily be found... The docs about translations could then even have a direct link to list all issues with that label.

**Comment by squidfunk** [comment]
We usually don't do feature-related labels. The label would have the exact same result as [searching for translation](https://github.com/squidfunk/mkdocs-material/issues?q=translation).
